spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc

```diff
@@ -459,7 +459,29 @@ When that happens, check the server log files for further details.
 [[dashboard-job-executions-steps-progress]]
 === Step Execution History
 
-Under *Step Execution History*, you can also view various metrics associated with the selected step, such as duration, read counts, write counts, and others.
+Under *Step Execution History*, you can also view various metrics associated with the selected step, such as duration, read counts, write counts, and others across all of its executions.
+For each metric there are 5 attributes that are available for each metric:
```

Comment:
**[NIT]** Redundant "For each metric..... for each metric". I think we can drop the trailing "for each metric:".

Reply:
Doh!   Good catch.